File: AspTask-EmployeeDirectory.UI/ClientApp/src/assets/employee.ts
export class Employee{
    
    firstname:string;
    lastname:string;
    preferredName:string;
    email:string;
    jobTitleID:number;
    officeID:number;
    departmentID:number;
    phoneNumber:number;
    skypeID:string;
    employeeID:number;
    employeeStatus:string;

    constructor(){
        this.firstname = '';
        this.lastname ='';
        this.preferredName = '';
        this.email = '';
        this.jobTitleID = 0;
        this.officeID = 0;
        this.departmentID = 0;
        this.phoneNumber = 0 ;
        this.skypeID = '';
        this.employeeID = 0;
        this.employeeStatus = '';
    }

}

File: AspTask-EmployeeDirectory.Models/Employee.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AspTask_EmployeeDirectory.Models
{
    public class Employee
    {
        public string Firstname { get; set; }
        public string Lastname { get; set; }
        public string Email { get; set; }
        public string DepartmentID { get; set; }
        public string JobTitleID { get; set; }
        public string OfficeID { get; set; }
        public long PhoneNumber { get; set; }
        public string SkypeID { get; set; }
        public string PreferredName { get; set; }
        public int EmployeeID { get; set; }
    }
}
